redist/org.goldendict.GoldenDict.appdata.xml

```diff
@@ -34,6 +34,6 @@
   </screenshots>
   <url type="homepage">http://goldendict.org</url>
   <update_contact>https://github.com/goldendict/goldendict</update_contact>
-  <launchable type="desktop-id">goldendict.desktop</launchable>
+  <launchable type="desktop-id">org.goldendict.GoldenDict</launchable>
```

Comment:
Shouldn't the `desktop-id` be `org.goldendict.GoldenDict.desktop`?

Reply:
Yeah, it should. It needs to be a filename.

Reply:
Yeah, annoying confusion...